metrics/integration/expression-tests/index-of/assert-array/test.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
All tests are copy-pasted from here https://github.com/maplibre/maplibre-gl-js/tree/main/test/integration/expression/tests/index-of